packages/cli-core/src/notices/notices.ts

```diff
@@ -0,0 +1,41 @@
+import z from 'zod';
+
+const noticePredicateSchema = z.discriminatedUnion('type', [
+  z.object({
+    type: z.literal('packageVersion'),
```

Comment:
I don't think the `node` version will be captured by this, if not, we should add that as a predicate. What about OS as well?

Reply:
good ideas, will do.